YouTube Player

mattleibow opened the pull request.

bholmes committed:
[YTPlayer] Adding an iOS binding for YouTube Player

The API defs have some TODOs that I need to review with the team
and work out when building the sample.

bholmes committed:
[YTPlayer] Adding an Android binding for YouTube Player

Need to figure out licensing before we package.

mattleibow committed:
Merge branch 'master' into youtube-player

mattleibow committed:
[YouTube] Updated the API and added samples

mattleibow committed:
[YouTube] Added the initial Android sample

mattleibow committed:
[YouTube] Clean up the sample code a bit

mattleibow committed:
[YouTube] Added a new iOS sample
 - added icons
 - added iOS component

bholmes commented on XPlat/YouTube.Player/ios/source/YouTube.Player.iOS/ApiDefinition.cs:
```diff
@@ -0,0 +1,241 @@
+	{
+		// Base Constructor
+		[Export("initWithFrame:")]
+		IntPtr Constructors(CGRect frame);
```
This should be `Constructor`?

bholmes committed:
[YouTube] Fixes for YouTube iOS component

 - Fixed some paths in the component.yaml files
 - Fixed a typo in APIDefinition file.
